controllers/admin/AdminImagesController.php

```diff
@@ -178,6 +192,51 @@ public function __construct()
                 ],
                 'submit' => ['title' => $this->trans('Save', [], 'Admin.Actions')],
             ],
+            'optimization' => [
+                'title' => $this->trans('Advanced optimization options', [], 'Admin.Design.Feature'),
+                'icon' => 'icon-picture',
+                'top' => '',
+                'bottom' => '',
+                'description' => $this->trans('Optimize bandwidth and disk usage.', [], 'Admin.Design.Help'),
+                'fields' => [
+                    'PS_IMAGEOPT_NO_ENLARGE' => [
+                        'type' => 'bool',
+                        'title' => $this->trans('Do not upscale images', [], 'Admin.Design.Feature'),
```

Comment:
The thing that I find very confusing here is the use of a sentence that starts with "Do not" combined with a Yes/No toggle button.

Is it: 
"**Yes, I don't want to** upscale images" and "**No, I want to** upscale images"? 

All the other labels are positive sentences. As a user, if I'm scanning the page, this is not clear at all.

Maybe it would be clearer to have "Upscale images" and the toggle set on "No". The help text could be something like "Keep this option disabled to optimize bandwidth and disk usage (...)".